src/components/CreateCard.vue

```diff
@@ -0,0 +1,166 @@
+<template>
+  <v-dialog v-model="active" persistent max-width="600">
+    <v-card>
+      <v-card-title class="headline">Crie sua nova lista de tarefas:</v-card-title>
+      <v-card-text>
+        <v-container>
+          <v-layout row>
+            <v-flex grow pa-1>
+              <v-text-field v-model="title" label="Insira o titulo da tarefa"></v-text-field>
+            </v-flex>
+          </v-layout>
+          <v-layout v-if="title" row wrap>
+            <v-flex xs12>
+              <v-radio-group v-model="radioGroup" row>
+                <v-radio
+                  v-for="(n,index) in optionList"
+                  :key="index"
+                  :label="`${n.text}`"
+                  :value="n.type"
+                ></v-radio>
+              </v-radio-group>
+            </v-flex>
+          </v-layout>
+          <v-layout v-if="title" row>
+            <v-flex grow pa-1>
+              <v-text-field v-model="text" label="Insira a frase da sua tarefa"></v-text-field>
+            </v-flex>
+            <v-flex shrink pa-1>
+              <v-btn @click="addingNewTask" icon>
+                <v-icon>mdi-plus</v-icon>
+              </v-btn>
+            </v-flex>
+          </v-layout>
+        </v-container>
+      </v-card-text>
+      <v-card-text>
+        <v-container>
+          <v-layout v-for="(task, index) in tasks" :key="index" row>
+            <v-flex grow pa-1>
+              <v-checkbox v-if="task.type === 'checkbox'" :label="task.text" disabled hide-details></v-checkbox>
+              <v-text-field v-else v-model="task.text" disabled></v-text-field>
+            </v-flex>
+            <v-flex shrink pa-1>
+              <v-btn @click="excludeTask(task.id)" icon>
+                <v-icon>close</v-icon>
+              </v-btn>
+            </v-flex>
+          </v-layout>
+        </v-container>
+      </v-card-text>
+      <v-snackbar v-model="snackbar" right color="error" :timeout="3000">
+        Preencha todos os dados para criar sua lista de tarefas
+        <v-btn dark flat @click="snackbar = false">
+          <v-icon>close</v-icon>
+        </v-btn>
+      </v-snackbar>
+      <v-card-actions>
+        <v-spacer></v-spacer>
+        <v-btn color="error" text @click="closeModal">cancelar</v-btn>
+        <v-btn color="success" text @click="saveAllTasks">salvar nova lista</v-btn>
+      </v-card-actions>
+    </v-card>
+  </v-dialog>
+</template>
+<script>
+export default {
+  props: {
+    active: Boolean,
+    task: Object
```

Comment:
When you have an object you should use an interface.

`Object as () => TaskInterface`